lib/l10n/intl_ca.arb

```diff
@@ -1001,5 +1001,6 @@
   "@discardChangesWarning": {
     "description": "Warning message shown when the user tries to close a route without saving the changes."
   },
-  "changelog": "Changelog"
+  "changelog": "Changelog",
```

Comment:
Regarding all translations: Did you add the new strings to the .arb files directly or did you create them in crowdin?

Reply:
I added them directly

Reply:
I see. I think it would make more sense to _not_ show a message when the button is disabled due to the current app state (not connectivity state), i.e. don't show a snackbar at all. Otherwise we would need a reason why the button is disabled. It should be evident to the user that the upload button is disabled because there is nothing to upload. So maybe remove the translations and the `disabledReason` property. Only show the "You're offline" snackbar when the user is offline and therefore the button is disabled. If disabled is true but the user is online, don't show anything.